frameworks/hdfs/tests/config.py

```diff
@@ -20,58 +22,48 @@
 DOCKER_IMAGE_NAME = "mesosphere/hdfs-client:2.6.4"
 KERBERIZED_CLIENT_IMAGE_NAME = "nvaziri/kerberized-hdfs-client:dev"
```

Comment:
Would it make sense to only specify this in the `json` file?

Reply:
Yeah it probably does, as this shouldn't need to be modified anyway.